.env

```diff
@@ -13,25 +14,28 @@ FCREPO_BASE_PATH=/hykudemo
 FCREPO_HOST=fcrepo
 FCREPO_PORT=8080
 FCREPO_REST_PATH=rest
+HYRAX_ACTIVE_JOB_QUEUE=good_job
```

Comment:
- [ ] should goodjob be the default?
- [ ] does the bin/worker script auto select based on this variable?